src/plugins/expression_reveal_image/public/expression_renderers/reveal_image_renderer.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React, { lazy } from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { I18nProvider } from '@kbn/i18n/react';
+import { ExpressionRenderDefinition, IInterpreterRenderHandlers } from 'src/plugins/expressions';
+import { withSuspense } from '../../../presentation_util/public';
+import { getRendererStrings } from '../../common/i18n';
+import { RevealImageRendererConfig } from '../../common/types';
+import './reveal_image.scss';
```

Comment:
Can we move this `.scss` import to be in the component, so that the css gets out of the main bundle.  I think that should knock down the main bundle size by a few k

Reply:
Done.